.github/actions/setup-go-mod-private/action.yaml

```diff
@@ -12,6 +12,11 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Setup Go
+      uses: actions/setup-go@v4
```

Comment:
```suggestion
      uses: actions/setup-go@v5
```

Reply:
will do in the next PR globally to all our actions to not trigger the CI once again on this PR 👍